src/Ilios/CoreBundle/Resources/config/services.yml

```diff
@@ -192,4 +192,8 @@ services:
     Ilios\CoreBundle\Entity\Manager\UserRoleManager:
         public: true
         arguments:
-            $class: 'Ilios\CoreBundle\Entity\UserRole'
\ No newline at end of file
+            $class: 'Ilios\CoreBundle\Entity\UserRole'
+    Ilios\MeSH\Parser:
+        public: true
+    Ilios\CoreBundle\Classes\MeshDescriptorSetTransmogrifier:
```

Comment:
Not needed. Since this isn't ever called like `$container->get('transmogrifier')` it doesn't need to be public.